sdks/python/setup.py

```diff
@@ -204,7 +204,7 @@ def get_version():
     'google-cloud-datastore>=1.7.1,<1.8.0',
     'google-cloud-pubsub>=0.39.0,<1.1.0',
     # GCP packages required by tests
-    'google-cloud-bigquery>=1.6.0,<=1.24.0',
+    'google-cloud-bigquery>=1.6.0,<=1.26.1',
```

Comment:
I would use  `<1.27.0` as an upper bound for this use case. Note that we also have https://github.com/apache/beam/pull/12198 in flight that further extends the upper bound.